include/zephyr/usb_c/usbc.h

```diff
@@ -109,6 +109,10 @@ enum usbc_policy_notify_t {
 	POWER_CHANGE_1A5,
 	/** Sink SubPower state a 5V / 3A */
 	POWER_CHANGE_3A0,
+	/** Sender Response Timeout */
```

Comment:
Do these belong in the "missing policy notifications" commit?

Reply:
The messing policy's commit is for the sample code, so I think it better to keep these in this PR.